plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -65,7 +65,7 @@ const PodContainerConfigSection = ({containerConfig, appConfig, onEditClick, ind
       </ConfigurationMapRow>
 
       {/* Resources */}
-      {fields.resources.cpus && (
+      {!!fields.resources.cpus && (
```

Comment:
I'm a little to late to the game, but let's please always try to avoid type casting esp. if it's changing the type representation as the result could be very different to the expected outcome. 

*Examples*
```
Boolean("0") === true
Boolean(0) === false
Boolean(Number("")) === true
Boolean(String(Number(null))) === true
Boolean(parseInt("1.1.1")) === true
Boolean(Number("1.1.1")) === false
// ...
```

If possible please always prefer using an expressive function/util that properly checks the values types. In this case we could use the `isNumberInRange` util as we actually want to check if it a number an if that number is greater than zero.

We might also want to add an eslint rule that at least warns if someone if casting type due to the named reasons.

/cc @dcos/frontend